### DISCUSSION TITLE
Seiten Customization

### DISCUSSION THREAD

**Discussion by synchronierer** [jzakotnik/openlibry#89] [Ideas]
Kann man die Seiten irgendwie anpassen?
Ich habe das mal versucht, indem ich an den *.tsx in /pages rumgebastelt habe, aber die Veränderungen werden auf der Seite nach einem Reload nicht angezeigt.

Ich hatte einfach nur mal versucht auf der Hauptseite den Namen unserer Schule einzutragen.

**Comment by jzakotnik** [comment]
Ja, das ist der richtige Weg. Nach dem anpassen am besten

- `npm run dev` starten und einmal lokal schauen ob es passt
- `npm run build` und `npm run start` um zu sehen wie es in prod aussieht oder den docker image/container nochmal bauen..

Falls Du dieses Repo clonest und die changes dort eincheckst kann ich auch schauen ob was verkehrt ist. Glückwunsch zu den ersten Steps in Typescript :-)

**Reply by synchronierer** [reply]
Das sind keine Schritte, das ist Gestolper... ich kenne mich grad mal bisschen mit der Apache-Welt aus und jetzt das. :-)

`npm run dev` führt bei mir zu sh: next: not found, zuvor bin ich mit `docker exec -it openlibryls_neu /bin/sh` rein.
`npm i run dev` ergibt [#########.........] \ idealTree: timing idealTree Completed in 1291ms und dann tut sich nix mehr.

**Reply by jzakotnik** [reply]
Ah, es war mir nicht klar, dass Du das im container ausführst. Würde ein `npm install` und dann ein `npm run dev` klappen? Das würde die Pakete (z.B. `next`) erst installieren

**Reply by synchronierer** [reply]
`npm install` liefert auch wieder so eine Fehlermeldung
[#########.........] \ idealTree: timing idealTree Completed in 711ms
Ich muss dann mit STRG+C abbrechen.

**Comment by jzakotnik** [comment]
Kann es sein dass der Node Server noch läuft in dem Container ? Dann klappt es ggf nicht .. geht `killall node` ? Und dann? 
ansonsten wäre noch eine Idee den node_modules folder zu löschen, `npm install` erzeugt ihn dann neu

**Reply by synchronierer** [reply]
`killall node` hat mich aus dem Container geschmissen, weil der damit gestoppt wird.
node_modules konnte ich löschen und `npm install` lief dann durch.
`npm run build` lief auch durch, aber die Seiten werden immer noch nicht anders dargestellt.

Wenn ich mit nano in die files schaue, steht meine Veränderung drin, aber im Browser ändert sich nichts.

**Comment by jzakotnik** [comment]
Das ist ja faszinierend... ist es ggf. noch im Cache,  bzw. im Inkognito Modus auch? der hot reload geht nur mit `npm run dev`..
